lib/ruby_indexer/lib/ruby_indexer/location.rb

```diff
@@ -44,5 +44,17 @@ def initialize(start_line, end_line, start_column, end_column)
       @start_column = start_column
       @end_column = end_column
     end
+
+    sig do
+      params(
+        another_location: T.any(Location, Prism::Location)
+      ).returns(T::Boolean)
+    end
+    def ==(another_location)
+      start_line == another_location.start_line &&
+        end_line == another_location.end_line &&
+        start_column == another_location.start_column &&
+        end_column == another_location.end_column
+    end
```

Comment:
not sure if this implementation is good enough or we should this classes' `from_prism_location` method.

Also wanted to mention that I was not able to find a unit test file for this class and I was wondering if I should create it.

Reply:
Implementation looks good. We didn't have a test because the implementation of this class is generally very simple and ends up being tested by the logic that uses it.

For example, in the reference finder tests, this will be tested by verifying that we do not add duplicate references for a class/module declaration.

Reply:
cool, that makes sense to me. Thanks for the clarification.